src/core/bridge.ts

```diff
@@ -149,6 +149,22 @@ export abstract class AbstractBridge {
       });
   }
 
+  static normalizeRequest(request: Request): Request {
+    Object.keys(request).forEach((section) => {
+      const callback = AbstractBridge[`normalize${section.charAt(0).toUpperCase() + section.slice(1)}`];
```

Comment:
Feel like this short be extracted from some kind of map exposed by the `AbstractBridge` class.

ie.

```
static get normalizers() {
  return {
    sort: AbstractBridge.normalizeSort,
  };
}
```

Reply:
This would also allow us to iterate over the `normalizers` keys, rather than the request keys.